tests/python/frontend/tflite/test_forward.py

```diff
@@ -2486,9 +2492,9 @@ def test_all_elemwise():
     _test_forward_elemwise_quantized(_test_maximum)
     _test_forward_elemwise(_test_minimum)
     _test_forward_elemwise_quantized(_test_minimum)
-    _test_forward_elemwise(_test_greater)
     _test_forward_elemwise(_test_squared_difference)
     _test_forward_elemwise(_test_greater_equal)
+    _test_forward_elemwise(_test_greater)
```

Comment:
Do we also need a call to `_test_forward_elemwise_quantized` to test the quantized version of "greater"?

Reply:
That is correct, got a patch in the pipes to get that in, a couple functions need tweaking for comparison operators like these.